packages/pilot/src/components/Operations/style.css

```
@@ -85,3 +85,12 @@
   width: 100%;
   z-index: 5;
 }
+
+.bold {
+  color: var(--color-light-steel-100);
+  font-weight: bold;
+}
+
+.title {
+  font-size: 14px;
```

Acho melhor tirar o `color: var(--color-light-steel-100);` de `.bold` e colocar aqui!
```suggestion
  font-size: 14px;
  color: var(--color-light-steel-100);
```

---

Feito!